script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -258,32 +286,33 @@ Akiha grumbles in frustration.
 [sha:634e873723c4479667ea985d406fb2bb208325c7]{
 -- Page 55, Offset 18398.
 -- 「いえ、何でもありません。それではごきげんよう。兄さんも遅刻などしないでくださいね」
-"It is nothing. Have a good day, Nii-san, and please try not to be late."
+"It's nothing, never mind. Have a good day, Nii-san, and please try not to be late."
 }
 [sha:df8cb078bb227e686ca181ccc3ceacb8d5b5df09]{
 -- Page 56, Offset 18399.
 -- 秋葉は琥珀さんと共に居間を後にした。
-Akiha leaves the living room with Kohaku-san.
+Akiha leaves the living room along with Kohaku-san.
 }
 [sha:fed59a538f6286ce190a8148456458a542a352e1]{
 -- Page 56, Offset 18400.
 -- ひとりになって改めて自問する。
-Now alone, I question myself once again.
+Now alone, I pose myself another question.
```

Comment:
Maybe drop the 'question' stuff entirely (because 'another question' isn't really  what's going on) and just go with something like:

> Now alone, I once again ask myself what's next.